### ISSUE TITLE
Add `check` subcommand to help with updates

### ISSUE THREAD

**Issue by LukasKalbertodt** [elan-ev/tobira#272]
I recently manually updated a deployment of Tobira. Overall I was pretty happy: I copied over the binary and just `systemctl restart tobira` which was done almost instantly. Of course there are to catches: (a) if DB migrations are necessary, that might take some time, and (b) if any configuration is incorrect with the new version, it won't start.

Solving (a) is super hard. We cannot start the migration while the old version of Tobira is still running. Or at least, it would require quite a bit of synchronization.

Solving (b) should be possible. I want a subcommand of tobira that basically verifies that Tobira can be started. This includes:
- Making sure the config file is valid
- Make sure all assets referenced in the config file exist
- Make sure the DB can automatically be migrated (if that's necessary at all)

So basically, if the subcommand says "all good", then I expect a `systemctl restart tobira` to always work without a problem.
